cmd/buildkitd/main_oci_worker.go

```diff
@@ -49,9 +55,13 @@ func init() {
 		},
 		cli.StringFlag{
 			Name:  "oci-worker-snapshotter",
-			Usage: "name of snapshotter (overlayfs or native)",
+			Usage: "name of snapshotter (overlayfs, native, etc.)",
 			Value: defaultConf.Workers.OCI.Snapshotter,
 		},
+		cli.StringFlag{
+			Name:  "oci-worker-proxy-snapshotter-address",
+			Usage: "use proxy snapshotter at specified address",
```

Comment:
With `unix://` or without?

Reply:
Thanks for your feedback. Explicitly described not to include `unix://` prefix and added small validation against the specified path.

Reply:
imho `*-address` should have protocol, `*-path` or `*-socket` doesn't need to.